Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -50,6 +50,11 @@ final class StringExtensionsTests: XCTestCase {
         XCTAssertEqual("Hello".firstCharacterAsString, "H")
     }
 
+    func testFirstCharacterUppercased() {
+        XCTAssertNotNil("hello world".firstCharacterUppercased)
```

Comment:
This isn't a useful test, since the return value of the method isn't optional. Better to test the empty string.